cli/spec/kontena/cli/stacks/list_command_spec.rb

```diff
@@ -60,4 +61,77 @@
       end
     end
   end
+
+  context "command output layout" do
+    before do
+      allow(client).to receive(:get).with('grids/test-grid/stacks').and_return({
+        'stacks' => []
+      })
+    end
+
+    it "renders NAME as 1st column" do
```

Comment:
This series of specs is maybe a bit overly strict. If there are any changes to the fields, this whole set will blow up.

There's also a helper for testing the table outputs, It works something like:

```ruby
expect{subject.run([])}.to output_table([
  %w(xyz abc dfg)
]).with_header(
  %w(NAME FOO LABELS)
)
```

(see `table_generator_spec.rb`)

Reply:
✅ RESOLVED